src/main/webapp/WEB-INF/pages/cart.jsp

```diff
@@ -0,0 +1,99 @@
+<%@ page contentType="text/html;charset=UTF-8" %>
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ taglib prefix="fmt" uri="http://java.sun.com/jsp/jstl/fmt" %>
+<%@ taglib prefix="tags" tagdir="/WEB-INF/tags" %>
+
+<jsp:useBean id="cart" type="com.es.phoneshop.model.cart.Cart" scope="request"/>
+<tags:master pageTitle="Cart">
+    <p>
+        Cart : ${cart}
+    </p>
+    <p>
+        <span class="success">
+                ${param.message}
+        </span>
+    </p>
+    <p>
+    <c:if test="${not empty errors}">
+        <div class="error">
+            There were some errors updating a cart.
+        </div>
+    </c:if>
+    </p>
+    <form method="post" action="${pageContext.servletContext.contextPath}/cart">
+        <table>
+            <thead>
+            <tr>
+                <td>Image</td>
+                <td>
+                    Description
+                </td>
+                <td class="quantity">
+                    Quantity
+                </td>
+                <td class="price">
+                    Price
+                </td>
+                <td>
+
+                </td>
```

Comment:
Please format the HTML code:
```

<table>
    <thead>
        <tr>
            <td>Image</td>
            <td>Description</td>
            <td class="quantity">Quantity</td>
            <td class="price">Price</td>
            <td></td>
```
Do we need empty TD tag?
More info: https://www.w3schools.com/html/html5_syntax.asp